dali/test/python/test_eager_operators.py

```diff
@@ -0,0 +1,196 @@
+        out2_data = out2.as_cpu() if isinstance(out2, tensors.TensorListGPU) else out2
+
+        check_batch(out1_data, out2_data, batch_size)
+
+
+def test_rotate_cpu():
+    compare_eager_with_pipeline('rotate', angle=25)
+
+
+def test_brightness_contrast_cpu():
+    compare_eager_with_pipeline('brightness_contrast')
+
+
+def test_hue_cpu():
+    compare_eager_with_pipeline('hue')
+
+
+def test_brightness_cpu():
+    compare_eager_with_pipeline('brightness')
+
+
+def test_contrast_cpu():
+    compare_eager_with_pipeline('contrast')
+
+
+def test_hsv_cpu():
+    compare_eager_with_pipeline('hsv')
+
+
+def test_color_twist_cpu():
+    compare_eager_with_pipeline('color_twist')
+
+
+def test_saturation_cpu():
+    compare_eager_with_pipeline('saturation')
+
+
+def test_shapes_cpu():
+    compare_eager_with_pipeline('shapes')
+
+
+def test_crop_cpu():
+    compare_eager_with_pipeline('crop', crop=(5, 5))
+
+
+def test_color_space_coversion_cpu():
+    compare_eager_with_pipeline('color_space_conversion',
+                                image_type=types.BGR, output_type=types.RGB)
+
+
+def test_cast_cpu():
+    compare_eager_with_pipeline('cast', dtype=types.INT32)
+
+
+def test_resize_cpu():
+    compare_eager_with_pipeline('resize', resize_x=50, resize_y=50)
+
+
+def test_gaussian_blur_cpu():
+    compare_eager_with_pipeline('gaussian_blur', window_size=5)
+
+
+def test_laplacian_cpu():
+    compare_eager_with_pipeline('laplacian', window_size=5)
+
+
+def test_crop_mirror_normalize_cpu():
+    compare_eager_with_pipeline('crop_mirror_normalize')
+
+
+def test_flip_cpu():
+    compare_eager_with_pipeline('flip', horizontal=True)
+
+
+def test_jpeg_compression_distortion_cpu():
+    compare_eager_with_pipeline('jpeg_compression_distortion', quality=10)
+
+
+def test_reshape_cpu():
+    new_shape = sample_shape.copy()
+    new_shape[0] //= 2
+    new_shape[1] *= 2
+    compare_eager_with_pipeline("reshape", shape=new_shape)
+
+
+def test_reinterpret_cpu():
+    compare_eager_with_pipeline("reinterpret", rel_shape=[0.5, 1, -1])
+
+
+def test_water_cpu():
+    compare_eager_with_pipeline("water")
+
+
+def test_sphere_cpu():
+    compare_eager_with_pipeline("sphere")
+
+
+def test_erase_cpu():
+    compare_eager_with_pipeline("erase", anchor=[0.3], axis_names="H",
+                                normalized_anchor=True, shape=[0.1], normalized_shape=True)
+
+
+def test_expand_dims_cpu():
+    compare_eager_with_pipeline("expand_dims", axes=1, new_axis_names="Z")
+
+
+def test_coord_transform_cpu():
+    M = [0, 0, 1,
+         0, 1, 0,
+         1, 0, 0]
+    compare_eager_with_pipeline("coord_transform", M=M, dtype=types.UINT8)
+
+
+def test_grid_mask_cpu():
+    compare_eager_with_pipeline("grid_mask", tile=51, ratio=0.38158387, angle=2.6810782)
+
+import torch
+
+def test_multi_paste_cpu():
+    compare_eager_with_pipeline("multi_paste", in_ids=torch.tensor([0, 1], dtype=torch.int32, device='cuda'), output_size=sample_shape)
+
```

Comment:
Do we really need torch here? can we use np?

Reply:
Of course. It was numpy, I changed it to torch tensor to check if it works with GPU data and forgot to change it back.